libs/utils/trace.py

```diff
@@ -394,6 +395,23 @@ def getTasks(self):
         """
         return self._tasks_by_pid.TaskName.to_dict()
 
+    def show(self):
+        """
+        Open the parsed trace using the most appropriate native viewer.
+
+        The native viewer depends on the specified trace format:
+        - ftrace: open using kernelshark
+        - systrace: open using a browser
+
+        In both cases the native viewer is assumed to be available in the host
+        machine.
+        """
+        if isinstance(self.ftrace, trappy.FTrace):
+            return os.popen("kernelshark {}".format(self.ftrace.trace_path))
```

Comment:
subprocess.call is more appropriate, and will not have issues with space and special chars in the file name